core/src/main/java/com/microsoft/applicationinsights/internal/heartbeat/HeartBeatProvider.java

```diff
@@ -133,10 +135,10 @@ public boolean addHeartBeatProperty(String propertyName, String propertyValue,
              payload.setPayloadValue(propertyValue);
              heartbeatProperties.put(propertyName, payload);
              isAdded = true;
-             InternalLogger.INSTANCE.trace("added heartbeat property");
+             InternalLogger.INSTANCE.trace("added heartbeat property %s - %s", propertyName, propertyValue);
```

Comment:
Hyphen is an interesting choice. Usually you see `%s:%s` or `%s=%s` to indicate key=value.

Reply:
This is nit: I think should be fine :)